scripts/api-clients-shell.py

```diff
@@ -10,6 +10,7 @@
   --api-token         Override for the API key (don't decrypt from dm-credentials)
   --search-api-url    Override the implicit SearchAPI URL
   --search-api-token  Override for the SearchAPI key (don't decrypt from dm-credentials)
+  --read-only         Only allow access to API calls that read data
```

Comment:
Can we have a short flag as well as a long flag for this?

```suggestion
  -r, --read-only         Only allow access to API calls that read data
```

Reply:
Great idea - I've just added a commit to do that.